hadoop-hdds/client/src/main/java/org/apache/hadoop/hdds/scm/storage/BlockDataStreamOutput.java

```diff
@@ -485,7 +503,7 @@ private void handleFlush(boolean close)
       // data since latest flush - we need to send the "EOF" flag
       executePutBlock(true, true);
     }
-    putBlockFuture.get().get();
+    CompletableFuture.allOf(putBlockFutures.toArray(EMPTY_FUTURE_ARRAY)).get();
```

Comment:
I think we need to handle the exceptions here

Reply:
handleFlush() already throws IOException, InterruptedException, ExecutionException.  should I specifically catch any here?